src/Microsoft.AspNetCore.Hosting/Microsoft.AspNetCore.Hosting.csproj

```diff
@@ -6,6 +6,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;hosting</PackageTags>
+    <LangVersion>latest</LangVersion>
```

Comment:
This should be done solution wide @natemcmaster any ideas?

Reply:
> This should be done solution wide @natemcmaster any ideas?

Add it to Directory.Build.props